proposals/per_policy_webhook_configurations.md

```diff
@@ -0,0 +1,235 @@
+# Meta
+[meta]: #meta
+- Name: Policy-based Webhook Configurations
+- Start Date: 2023-12-12
+- Update data (optional):
+- Author(s): @realshuting
+
+# Table of Contents
+[table-of-contents]: #table-of-contents
+- [Meta](#meta)
+- [Table of Contents](#table-of-contents)
+- [Overview](#overview)
+- [Definitions](#definitions)
+- [Motivation](#motivation)
+- [Proposal](#proposal)
+- [Implementation](#implementation)
+- [Migration (OPTIONAL)](#migration-optional)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Prior Art](#prior-art)
+- [Unresolved Questions](#unresolved-questions)
+- [CRD Changes (OPTIONAL)](#crd-changes-optional)
+
+# Overview
+[overview]: #overview
+
+Configure Kyverno resource webhook configurations based on policy settings.
```

Comment:
Perhaps we should briefly describe the current scheme and why that is insufficient?

Reply:
Updated. 

I believe the benefits under `Motivation` section point out current issues. Does that make sense to you?